hasura/graphql-engine
# server: fix issue with tracking custom functions that return `SETOF` materialized view (close #5294)

fixes #5294 

### Description
This PR adds a fix to help track custom functions that return a `setof` elements from a materialized view.

### Affected components



### Steps to test and verify

- Create a Materialized View
- Create a custom function that returns `setof` materialized view
- Click on track function on `Data` page to see if this custom function is tracked.
- Run Graphql queries from GraphiQL editor to verify,.

### Server checklist


#### Catalog upgrade

Does this PR change Hasura Catalog version?

#### Metadata


Does this PR add a new Metadata feature?

#### GraphQL

#### Breaking changes
